webapp/components/Badges.vue

```diff
@@ -10,17 +10,21 @@
       :key="badge.key"
       class="hc-badge-container"
     >
-      <img
+      <hc-image
         :title="badge.key"
-        :src="badge.icon"
+        :image-props="{ src: badge.icon}"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-07T16:46:17Z" title="Tuesday, May 7th 2019, 6:46:17 pm +02:00">May 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -10,22 +10,31 @@
       :key="badge.key"
       class="hc-badge-container"
     >
-      <img
+      <hc-image
         :title="badge.key"
-        :src="badge.icon"
+        :image-props="imageProps(badge.icon)"
```


```suggestion
        :image-props="{src: badge.icon}"
```
@aonomike that does not work?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-05-07T23:27:22Z" title="Wednesday, May 8th 2019, 1:27:22 am +02:00">May 8, 2019</time>_

---


when I was coding with him @roschaefer this wasn't working because it is calling `this.imageProps` and that was throwing an error, but I haven't pulled it down and played around with it, so I could be off base

Reply:
> [<img alt="aonomike" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/aonomike) **Authored by [aonomike](https://github.com/aonomike)**
_<time datetime="2019-05-08T12:49:48Z" title="Wednesday, May 8th 2019, 2:49:48 pm +02:00">May 8, 2019</time>_

---


Yes this simplifies the code. I have refactored with this. Thanks @roschaefer @mattwr18